mkdocs.yml

```
@@ -169,6 +169,7 @@ nav:
     -   Using MkDocs macros: extensions/macros_doc.md
 -   Impact:
     -   impact/index.md
+    -   impact/dashboards.md
```

```suggestion
    -   impact/measuring_BIDS_impact.md
```

Rename to make rendering/reading of markdown yaml more clear later.